common/men_at_arms_types/wc_cultural_maa_types.txt

```diff
@@ -0,0 +1,1708 @@
+		massive_aerial = 1
+	}
+
+	buy_cost = { gold = cost_135_recruitment_cost }
+	low_maintenance_cost = { gold = cost_135_low_maint_cost }
+	high_maintenance_cost = { gold = cost_135_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = gyrocopter
+}
+
+headhunter = {
+	type = archers
+	
+	damage = 35
+	toughness = 20
+	pursuit = 15
+	screen = 10
+	
+	terrain_bonus = {
+	}
+	
+	counters = {
+		skirmishers = 1
+		light_aerial = 1
+	}
+	
+	buy_cost = { gold = cost_90_recruitment_cost }
+	low_maintenance_cost = { gold = cost_90_low_maint_cost }
+	high_maintenance_cost = { gold = cost_90_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = headhunter
+}
+
+hexer = {
+	type = caster
+	
+	damage = 60
+	toughness = 15
+	pursuit = 15
+	screen = 10
+	
+	terrain_bonus = {
+		jungle = { damage = 15 toughness = 15 pursuit = 5 }
+		plains = { damage = 10 pursuit = 5 }
+		forest = { damage = 15 toughness = 15 pursuit = 5}
+	}
+	
+	counters = {
+		light_aerial = 1
+		massive_infantry = 1
+		heavy_infantry = 1
+	}
+	
+	buy_cost = { gold = cost_105_recruitment_cost }
+	low_maintenance_cost = { gold = cost_105_low_maint_cost }
+	high_maintenance_cost = { gold = cost_105_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = hexer
+}
+
+huntress = {
+	type = light_cavalry
+	
+	damage = 35
+	toughness = 25
+	pursuit = 25
+	screen = 20
+	
+	terrain_bonus = {
+		plains = { damage = 20 }
+		desert = { damage = 10 }
+		forest = { damage = 15 toughness = 5 }
+		jungle = { damage = 10 toughness = 5 }
+	}
+
+	counters = {
+		archers = 1
+		caster = 1
+	}
+
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = huntress
+}
+
+magi = {
+	type = caster
+	
+	damage = 40
+	toughness = 40
+	pursuit = 10
+	screen = 10
+	
+	terrain_bonus = {
+		hills = { damage = 15 toughness = 10 }
+		desert = { damage = 20 toughness = 10 }
+		drylands = { damage = 15 toughness = 10 }
+	}
+	
+	counters = {
+		light_aerial = 1
+		massive_infantry = 1
+		heavy_infantry = 1
+	}
+	
+	buy_cost = { gold = cost_110_recruitment_cost }
+	low_maintenance_cost = { gold = cost_110_low_maint_cost }
+	high_maintenance_cost = { gold = cost_110_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = magi
+}
+
+marauder = {
+	type = light_cavalry
+	
+	damage = 35
+	toughness = 25
+	pursuit = 30
+	screen = 25
+	
+	terrain_bonus = {
+		plains = { damage = 20 }
+		drylands = { damage = 25 }
+		desert = { damage = 20 }
+	}
+
+	counters = {
+		archers = 1
+		pikemen = 1
+	}
+
+	buy_cost = { gold = cost_110_recruitment_cost }
+	low_maintenance_cost = { gold = cost_110_low_maint_cost }
+	high_maintenance_cost = { gold = cost_110_high_maint_cost }
```

Comment:
Just realised in the spreadsheet that these are a little too powerful, maybe increase cost by 10?